README.md

```diff
@@ -0,0 +1,8 @@
+# Meeting room dashboard
+
+## Docker usage example
+### Build
+`docker build . --tag meeting-room-dashboard:1.0 --tag meeting-room-dashboard:latest`
+
+### Run
+`docker run -p 80:80 -d -e MRD_JSON_KEY="<cred_path>.json" -e MRD_PERSON="<impersonated_mail_address>" -e MRD_USER="<basicauth_usr>" -e MRD_PASSWORD="<basicauth_pw>" meeting-room-dashboard`
```

Comment:
for readme it's ok, later for production:

once you create a service:

```
docker service create \
   -e ENV_VARIABLE="value" \
   -e ANOTHER_VARIABLE="another value" \
   --replicas 1 \
   --update-parallelism=2 --update-delay=10s \
   --name meeting-room-dashboard \
   -p 9101:80/tcp \
   meeting-room-dashboard:YOUR_TAG
```

after that, you have to update the service only to release a new version:

```
docker service update --image=meeting-room-dashboard:NEWER_YOUR_TAG meeting-room-dashboard
```

Done :D